apps/api/src/bootstrap.ts

```diff
@@ -19,6 +19,8 @@ import { ResponseInterceptor } from './app/shared/framework/response.interceptor
 import { RolesGuard } from './app/auth/framework/roles.guard';
 import { SubscriberRouteGuard } from './app/auth/framework/subscriber-route.guard';
 import { validateEnv } from './config/env-validator';
+import { getOTELSDK } from '@novu/application-generic/build/main/tracing/tracing';
+import * as packageJson from '../package.json';
```

Comment:
we had some issues with this pattern before, so maybe let's remove it and hardcode the name in the `getOTELSDK` function call?

Reply:
Can you point me to thous issues?

I am doing the same thing with the logging PR.

Reply:
https://discord.com/channels/895029566685462578/1045289346598703164/1073211111874248754

Reply:
It was actually another thing that causing this previously. So I believe it should be a problem id the API is building succesfully